docs/application/web/get-started/overview.md

```diff
@@ -34,3 +34,17 @@ HTML5, CSS, and JavaScript:
 The first application examples show a simple application with a basic UI
 and minimal functionality. Get familiar with the example first, and
 afterwards you can use it as a basis to build more complex applications.
+
+# Get Started with Richness on Your Web Applications
```

Comment:
The overview page introduces contents in profile order: Wearable, TV, and Mobile. 
When I see screenshot your new documents, there are mobile screenhosts.
So...you have to put your new document as the last item of the list.

The heading 1 same as H1 tag is used for the page title.
Please DON"T USE heading1 for the section title.

Moreover, please see how this page is shown on the screen: 
https://github.com/Samsung/tizen-docs/blob/acafc3cb0088c78641b547a67f80dc61f3a9793a/docs/application/web/get-started/overview.md.

> NOTE
> Please click **File changed** > **...** > **View file** before submiting a PR.

Reply:
The reason I separated this from others is that it's not web application exactly.
It is a kind of extension which works over web applications.
Is that okay to keep the current with H2 for 'Get Started with Richness on Your Web Applications' ?

Reply:
I resolve this because, in the following conversation, you agreed on the title I suggested.